threedi_api_qgis_client/widgets/wizard.py

```diff
@@ -505,6 +528,28 @@ def load_csv(self):
         self.dwf_upload.setText(filename)
         self.dwf_timeseries = values
 
+    def handle_dwf_laterals_header(self, dwf_laterals_list, log_error=True):
+        """Fetching first DWF lateral row and handling potential header."""
```

Comment:
The fist line of a docstring should be a command. "Return the first DFW lateral and be quick about it!!!" :-)
So rather "Fetch first .... and handle header" than the descriptive "Fetching... handling..."

Reply:
Done.